bazel/bazel.go

```diff
@@ -23,13 +23,58 @@ import (
 	"io"
 	"os"
 	"os/exec"
+	"path/filepath"
 	"strings"
 
 	blaze_query "github.com/bazelbuild/bazel-watcher/third_party/bazel/master/src/main/protobuf"
 	"github.com/golang/protobuf/proto"
 )
 
-var bazelPath = flag.String("bazel_path", "bazel", "Path to the bazel binary to use for actions")
+var bazelPathFlag = flag.String("bazel_path", "", "Path to the bazel binary to use for actions")
+
+func findBazel() (string) {
+	if len(*bazelPathFlag) > 0 {
+		return *bazelPathFlag
+	} else {
+		path, err := exec.LookPath("bazel")
+		if err == nil {
+			return path
+		}
+
+		// No bazel binary is in the $PATH
+		// When running from the @bazel/ibazel npm package, our binary is
+		// /DIR/node_modules/@bazel/ibazel/bin/$ARCH/ibazel
+		// We can find bazel in
+		// /DIR/node_modules/@bazel/bazel-$ARCH/bazel-0.28.0-linux-x86_64
+		fmt.Printf("didn't find bazel in the $PATH, %v\n", os.Args[0])
+		s := strings.Split(filepath.ToSlash(os.Args[0]), "/")
```

Comment:
I'm curious if you have thoughts on how to write a test for the stuff north of here? I guess you could make a synthetic FS and set the path to contain it, but that feels not great.

Reply:
I refactored it a bit so the conditional logic is brain-dead simple. I think this means we just don't try to test these two conditions.